src/test/java/game/map/MapTest.java

```diff
@@ -107,4 +107,49 @@ void shouldThrowExceptionWhenRemovingPlayerThatDoesNotExist() {
             map.removePlayer(playerName);
         }, "No player exists with this name");
     }
+
+    @Test
+    void shouldAssignCountries() {
```

Comment:
In this method I don't see anywhere you are calling the `assignCountries` method in Map. This doesn't look like a valid test to me. 

I can write one unit test if you are not familiar with tests. You can add other tests similar to that. Is that okay?